mender-convert-package

```diff
@@ -202,6 +202,13 @@ disk_write_at_offset "${output_dir}/data.img" "${sdimg_path}" "${data_start}"
 log_info "Performing platform specific package operations (if any)"
 platform_package
 
+# Create bmap index
+index_fs_type=""
+if [ "${MENDER_USE_BMAP}" == "y" ] && [ -n "$(type -path bmaptool)" ]; then
+    index_fs_type=".sdimg.bmap"
+    run_and_log_cmd "/usr/bin/bmaptool create ${sdimg_path} > ${sdimg_path}.bmap"
+fi
+
```

Comment:
Actually we do not need 3rd party images in the `IMAGE_FSTYPES` variable for the test. So this can be simplified to:

```
if [ "${MENDER_USE_BMAP}" == "y" ]; then
    run_and_log_cmd "/usr/bin/bmaptool create ${sdimg_path} > ${sdimg_path}.bmap"
fi
```

Reply:
I assume you still want the check for presence of bmaptool though right?

Reply:
Yes, that would be great.

Reply:
OK.  This is ready to go.